### PR TITLE
Sba 119

### PR DESCRIPTION
**Pull request by Novice052**
I do want to be able to serve static files, however, the base_generic.html can't find the 'static' folder.  Would you please take a look?

The rest of it is just groundwork and will be elaborated.

### CODE REVIEW

File: build_automation/build_automation/settings.py [hunk 1]
```diff
@@ -122,8 +122,16 @@
 # https://docs.djangoproject.com/en/2.0/howto/static-files/
 
 STATIC_URL = '/static/'
+STATIC_ROOT = ''
+STATICFILES_DIRS = [
+    os.path.join(BASE_DIR, "static"),
+    '/build_automation/static/',
```
Comment (pradeepsixer) [inline build_automation/build_automation/settings.py]: This line (no 128) should be deleted, as there would be no such path on file systems.

File: build_automation/requirements/dev.txt [hunk 2]
```diff
@@ -1,2 +1,3 @@
+environ
```
Comment (pradeepsixer) [inline build_automation/requirements/dev.txt]: Please give a reference to where this package is used in the project (as a follow up comment to this review).

File: build_automation/build_automation/settings.py [hunk 3]
```diff
@@ -122,8 +122,16 @@
 # https://docs.djangoproject.com/en/2.0/howto/static-files/
 
 STATIC_URL = '/static/'
+STATIC_ROOT = ''
```
Comment (pradeepsixer) [inline build_automation/build_automation/settings.py]: If we're not going to use this, can't we safely delete this line?

**Review by pradeepsixer — CHANGES_REQUESTED** [review]
few minor changes are needed. everything else look good to me.

File: build_automation/build_automation/settings.py (hunk 1 above)
Comment (Novice052) [reply build_automation/build_automation/settings.py]: done

File: build_automation/requirements/dev.txt (hunk 2 above)
Comment (Novice052) [reply build_automation/requirements/dev.txt]: This is in the master and the package wasn't loaded.  Do we not need it?

File: build_automation/build_automation/settings.py (hunk 3 above)
Comment (Novice052) [reply build_automation/build_automation/settings.py]: I need to look at this.

**Review by pradeepsixer — APPROVED** [review]

**Pull request merged**